src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -73,6 +74,25 @@ const VirtualAccOpening = (() => {
                 },
             })
             .setVisibility(1);
+            
+        const get_selected_value = document.getElementById('select2-residence-container').getAttribute('title');
+        const residence_dropdown = document.getElementById('residence');
+
+        if (!isEuCountry(get_selected_value)) {
+            email_consent.classList.add('hide-checkbox');
+            consent_checkbox.classList.add('hide-checkbox');
```

Comment:
why not just email_consent.setVisibility(0) and consent_checkbox.setVisibility(0), then we don't need to add a new class or change our form manager behaviour

Reply:
The reason I add this class is not only to hide the checkbox, but mainly to see specifically in the form manager when a checkbox has this specific class. When a hidden checkbox has this class, instead of sending 0 (since it's unchecked) it will send a 1 (even when it's unchecked). The reason behind this, is that Ashkan mentioned that the checkbox has to still send the value 1, when it's hidden after selecting a non-EU country.